dags.yaml

```diff
@@ -1158,3 +1158,24 @@ bqetl_kpis_shredder:
   tags:
     - impact/tier_3
     - repo/bigquery-etl
+bqetl_default:
+  default_args:
+    depends_on_past: false
+    email:
+      - telemetry-alerts@mozilla.com
+    email_on_failure: true
+    email_on_retry: false
+    end_date: null
+    owner: telemetry-alerts@mozilla.com
+    retries: 2
+    retry_delay: 30m
+    start_date: '2023-09-01'
+  description: This is a default DAG to schedule tasks with lower business impact
+    or that don't require a new or existing DAG. Queries can be automatically scheduled
+    in this DAG using the option --use_default_dag. See [related documentation in
```

Comment:
@lucia-vargas-a this mentions the `--use_default_dag` option which got removed.

Reply:
Thank you! Follow up [PR-4220](https://github.com/mozilla/bigquery-etl/pull/4220).